content/news/2023.pml

```diff
@@ -54,6 +54,29 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item" id="summit-2023-cfp">
+    20 February 2023: <a href="../Foundation/Summit/2023/cfp">Call For Presentations</a> for the seL4 Summit 2023
```

Comment:
Also add a link to the summit page, ie 
<a href="../Foundation/Summit/2023/cfp">Call For Presentations</a> for the <a href="blah"> seL4 Summit 2023</a>